src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -323,6 +382,13 @@ private string GetTemplateParameterErrorsMessage(ITemplateInfo templateInfo, out
             return null;
         }
 
+        private string GetLanguageMismatchErrorMessage(string inputLanguage)
+        {
+            string invalidLanguageErrorText = LocalizableStrings.InvalidTemplateParameterValues;
+            invalidLanguageErrorText += Environment.NewLine + string.Format(LocalizableStrings.InvalidParameterDetail, "-lang", inputLanguage, "--language");
```

Comment:
Let's use full switch names when mentioning them in output messages (unless we're using the user specified form)